src/Recoil__State.re

```diff
@@ -0,0 +1,7 @@
+type t = string;
+[@bs.inline]
+let loading = "loading";
+[@bs.inline]
+let hasValue = "hasValue";
+[@bs.inline]
+let hasError = "hasError";
```

Comment:
I think we can drop this file given it's already declared in `Recoil__Loadable`

Reply:
Oh indeed sorry I forgot to delete it